bciers/apps/dashboard/auth/auth.config.ts

```diff
@@ -75,7 +75,7 @@ export default {
   ],
   secret: `${process.env.NEXTAUTH_SECRET}`,
   pages: {
-    error: "../auth/error", // Error code passed in query string as ?error=
+    error: "/auth/error", // Error code passed in query string as ?error=
```

Comment:
@Sepehr-Sobhani just flagging for you that I added this commit (very sneaky of me, I know). I've been having issues trying to get the branch to run locally (not because of your work, just because of our recent cycling of Keycloak secrets). I keep getting an "Invalid client secret" error in my console. But on the frontend, it was trying to redirect me to "localhost:3000/../auth/error" which was then raising another error for the invalid URL. This fixes it.

Reply:
Thanks for the fix 🙏